inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/decisions/AdvancedDecisionType.java

```
@@ -0,0 +1,77 @@
+package uk.ac.ucl.rits.inform.informdb.decisions;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import uk.ac.ucl.rits.inform.informdb.TemporalCore;
+import uk.ac.ucl.rits.inform.informdb.annotation.AuditTable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.time.Instant;
+
+/**
+ * Advanced decisions can be different in nature and therefore are distinguished by their type. Each of these types is
+ * identified by a care code and name, e.g. "DNACPR" with care code "COD4".
+ * @author Anika Cawthorn
+ */
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+@AuditTable
+public class AdvancedDecisionType extends TemporalCore<AdvancedDecisionType, AdvancedDecisionTypeAudit> {
```

I don't know if we're expecting these to ever change over time, might be worth just having the type without auditing, not sure it needs validFrom or storedFrom.

We can throw an exception in emap core if the name changes for a care code to ensure this is true

---

Sounds good to me, I was wondering about it at the time ...